src/classes/region.h

```diff
@@ -44,14 +47,20 @@ class Region
 
         // Initialise 3D map and determine valid voxels
         voxelMap_.initialise(nVoxels_.x, nVoxels_.y, nVoxels_.z);
-        for (auto x = 0; x < nVoxels_.x; ++x)
-            for (auto y = 0; y < nVoxels_.y; ++y)
-                for (auto z = 0; z < nVoxels_.z; ++z)
-                    voxelMap_[{x, y, z}] = {
-                        Vec3<int>(x, y, z),
-                        voxelCheckFunction(cfg, box_->getReal({(x + 0.5) * voxelSizeFrac_.x, (y + 0.5) * voxelSizeFrac_.y,
-                                                               (z + 0.5) * voxelSizeFrac_.z}))};
 
+        // Setup iterator for voxel map
+        Array3DIterator iterator(nVoxels_.x, nVoxels_.y, nVoxels_.z);
+
+        // Iterate voxels in parallel
+        dissolve::for_each_triplet(
+            ParallelPolicies::par, iterator.begin(), iterator.end(),
```

Comment:
```suggestion
        // Iterate voxels in parallel
        dissolve::for_each_triplet(
            ParallelPolicies::par, voxelMap.beginIndices(), voxelMap.endIndices(),
```

Make our intentions a bit more clear